src/box/iproto_constants.c

```diff
@@ -317,3 +317,40 @@ const char *vy_row_index_key_strs[VY_ROW_INDEX_KEY_MAX] = {
 	NULL,
 	"row index",
 };
+
+struct iproto_constant iproto_flag_constants[] = {
```

Comment:
I would make these things as `const struct iproto_constant`.

Reply:
Done. I did the same for arrays' sizes.
<details>
<summary>diff</summary>

```diff
diff --git a/src/box/iproto_constants.c b/src/box/iproto_constants.c
index 569ccf3f7..1fc05f3a6 100644
--- a/src/box/iproto_constants.c
+++ b/src/box/iproto_constants.c
@@ -318,39 +318,41 @@ const char *vy_row_index_key_strs[VY_ROW_INDEX_KEY_MAX] = {
 	"row index",
 };
 
-struct iproto_constant iproto_flag_constants[] = {
+const struct iproto_constant iproto_flag_constants[] = {
 	IPROTO_FLAGS(IPROTO_CONSTANT_MEMBER)
 };
 
-size_t iproto_flag_constants_size = lengthof(iproto_flag_constants);
+const size_t iproto_flag_constants_size = lengthof(iproto_flag_constants);
 
-struct iproto_constant iproto_key_constants[] = {
+const struct iproto_constant iproto_key_constants[] = {
 	IPROTO_KEYS(IPROTO_CONSTANT_MEMBER)
 };
 
-size_t iproto_key_constants_size = lengthof(iproto_key_constants);
+const size_t iproto_key_constants_size = lengthof(iproto_key_constants);
 
-struct iproto_constant iproto_metadata_key_constants[] = {
+const struct iproto_constant iproto_metadata_key_constants[] = {
 	IPROTO_METADATA_KEYS(IPROTO_CONSTANT_MEMBER)
 };
 
-size_t iproto_metadata_key_constants_size =
+const size_t iproto_metadata_key_constants_size =
 	lengthof(iproto_metadata_key_constants);
 
-struct iproto_constant iproto_ballot_key_constants[] = {
+const struct iproto_constant iproto_ballot_key_constants[] = {
 	IPROTO_BALLOT_KEYS(IPROTO_CONSTANT_MEMBER)
 };
 
-size_t iproto_ballot_key_constants_size = lengthof(iproto_ballot_key_constants);
+const size_t iproto_ballot_key_constants_size =
+	lengthof(iproto_ballot_key_constants);
 
-struct iproto_constant iproto_type_constants[] = {
+const struct iproto_constant iproto_type_constants[] = {
 	IPROTO_TYPES(IPROTO_CONSTANT_MEMBER)
 };
 
-size_t iproto_type_constants_size = lengthof(iproto_type_constants);
+const size_t iproto_type_constants_size = lengthof(iproto_type_constants);
 
-struct iproto_constant iproto_raft_keys_constants[] = {
+const struct iproto_constant iproto_raft_keys_constants[] = {
 	IPROTO_RAFT_KEYS(IPROTO_CONSTANT_MEMBER)
 };
 
-size_t iproto_raft_keys_constants_size = lengthof(iproto_raft_keys_constants);
+const size_t iproto_raft_keys_constants_size =
+	lengthof(iproto_raft_keys_constants);
diff --git a/src/box/iproto_constants.h b/src/box/iproto_constants.h
index dd183268d..dda16860f 100644
--- a/src/box/iproto_constants.h
+++ b/src/box/iproto_constants.h
@@ -71,10 +71,10 @@ enum {
 ENUM(iproto_flag, IPROTO_FLAGS);
 
 /** Constants generated from IPROTO_FLAGS. */
-extern struct iproto_constant iproto_flag_constants[];
+extern const struct iproto_constant iproto_flag_constants[];
 
 /** Size of iproto_flag_constants. */
-extern size_t iproto_flag_constants_size;
+extern const size_t iproto_flag_constants_size;
 
 #define IPROTO_KEYS(_)							\
 	_(IPROTO_REQUEST_TYPE, 0x00)					\
@@ -204,10 +204,10 @@ ENUM(iproto_key, IPROTO_KEYS);
 static_assert(iproto_key_MAX <= 0x80, "iproto_key_MAX must be <= 0x80");
 
 /** Constants generated from IPROTO_KEYS. */
-extern struct iproto_constant iproto_key_constants[];
+extern const struct iproto_constant iproto_key_constants[];
 
 /** Size of iproto_key_constants. */
-extern size_t iproto_key_constants_size;
+extern const size_t iproto_key_constants_size;
 
 /**
  * Keys, stored in IPROTO_METADATA. They can not be received
@@ -225,10 +225,10 @@ extern size_t iproto_key_constants_size;
 ENUM(iproto_metadata_key, IPROTO_METADATA_KEYS);
 
 /** Constants generated from IPROTO_METADATA_KEYS. */
-extern struct iproto_constant iproto_metadata_key_constants[];
+extern const struct iproto_constant iproto_metadata_key_constants[];
 
 /** Size of iproto_metadata_key_constants. */
-extern size_t iproto_metadata_key_constants_size;
+extern const size_t iproto_metadata_key_constants_size;
 
 #define IPROTO_BALLOT_KEYS(_)						\
 	_(IPROTO_BALLOT_IS_RO_CFG, 0x01)				\
@@ -244,10 +244,10 @@ extern size_t iproto_metadata_key_constants_size;
 ENUM(iproto_ballot_key, IPROTO_BALLOT_KEYS);
 
 /** Constants generated from IPROTO_BALLOT_KEYS. */
-extern struct iproto_constant iproto_ballot_key_constants[];
+extern const struct iproto_constant iproto_ballot_key_constants[];
 
 /** Size of iproto_ballot_key_constants. */
-extern size_t iproto_ballot_key_constants_size;
+extern const size_t iproto_ballot_key_constants_size;
 
 static inline uint64_t
 iproto_key_bit(unsigned char key)
@@ -373,10 +373,10 @@ extern const unsigned char iproto_key_type[iproto_key_MAX];
 ENUM(iproto_type, IPROTO_TYPES);
 
 /** Constants generated from IPROTO_TYPES. */
-extern struct iproto_constant iproto_type_constants[];
+extern const struct iproto_constant iproto_type_constants[];
 
 /** Size of iproto_type_constants. */
-extern size_t iproto_type_constants_size;
+extern const size_t iproto_type_constants_size;
 
 /** Convenient synonims for reserved members of iproto_type. */
 enum {
@@ -404,10 +404,10 @@ extern const char *iproto_type_strs[];
 ENUM(iproto_raft_keys, IPROTO_RAFT_KEYS);
 
 /** Constants generated from IPROTO_RAFT_KEYS. */
-extern struct iproto_constant iproto_raft_keys_constants[];
+extern const struct iproto_constant iproto_raft_keys_constants[];
 
 /** Size of iproto_raft_keys_constants. */
-extern size_t iproto_raft_keys_constants_size;
+extern const size_t iproto_raft_keys_constants_size;
 
 /**
  * Returns IPROTO type name by @a type code.
diff --git a/src/box/iproto_features.c b/src/box/iproto_features.c
index 57eb9918b..b5724381f 100644
--- a/src/box/iproto_features.c
+++ b/src/box/iproto_features.c
@@ -54,11 +54,12 @@ mp_decode_iproto_features(const char **data, struct iproto_features *features)
 	return 0;
 }
 
-struct iproto_constant iproto_feature_id_constants[] = {
+const struct iproto_constant iproto_feature_id_constants[] = {
 	IPROTO_FEATURES(IPROTO_CONSTANT_MEMBER)
 };
 
-size_t iproto_feature_id_constants_size = lengthof(iproto_feature_id_constants);
+const size_t iproto_feature_id_constants_size =
+	lengthof(iproto_feature_id_constants);
 
 void
 iproto_features_init(void)
diff --git a/src/box/iproto_features.h b/src/box/iproto_features.h
index 4d9138998..755fde7a6 100644
--- a/src/box/iproto_features.h
+++ b/src/box/iproto_features.h
@@ -54,10 +54,10 @@ extern "C" {
 ENUM(iproto_feature_id, IPROTO_FEATURES);
 
 /** Constants generated from IPROTO_FEATURES. */
-extern struct iproto_constant iproto_feature_id_constants[];
+extern const struct iproto_constant iproto_feature_id_constants[];
 
 /** Size of iproto_feature_id_constants. */
-extern size_t iproto_feature_id_constants_size;
+extern const size_t iproto_feature_id_constants_size;
 
 /**
  * IPROTO protocol feature bit map.
```
</details>